microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIModelFilePropsHealthTest.java

```diff
@@ -0,0 +1,100 @@
+package org.jboss.eap.qe.microprofile.health.integration;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.arquillian.api.ServerSetupTask;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+/**
+ * MP FT integration will be tested relying on FS based MP Config
+ * MP Config properties are provided by file-props model option - values are stored in files on FS.
+ * Files are backed-up before every test and restored after.
+ * Purpose of the class is to configure MP Config. Superclass is responsible for test execution.
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+@ServerSetup({ MicroProfileFTSetupTask.class, FailSafeCDIModelFilePropsHealthTest.SetupTask.class })
+public class FailSafeCDIModelFilePropsHealthTest extends FailSafeCDIHealthBaseTest {
+
+    private byte[] liveBytes;
+    private byte[] readyBytes;
+    private byte[] inMaintenanceBytes;
+    private byte[] readyinMaintenanceBytes;
```

Comment:
Isn't this one supposed to be `readyInMaintenanceBytes` (capital "i" in "In")? I see you're using this notation basically everywhere - e.g. as in `readyInMaintenanceFile`.